doc/develop/test/twister.rst

```diff
@@ -505,14 +505,36 @@ harness_config: <harness configuration options>
         Only one fixture can be defined per testcase and the fixture name has to
         be unique across all tests in the test suite.
 
+.. _pytest_root:
+
     pytest_root: <list of pytest testpaths> (default pytest)
-        Specify a list of pytest directories, files or subtests that need to be executed
-        when test case begin to running, default pytest directory is pytest.
-        After pytest finished, twister will check if this case pass or fail according
-        to the pytest report.
+        Specify a list of pytest directories, files or subtests that need to be
+        executed when test case begin to running, default pytest directory is
+        pytest. After pytest finished, twister will check if this case pass or
+        fail according to the pytest report. List of exemplary and valid
+        pytest roots is presented below:
```

Comment:
```suggestion
        Specify a list of pytest directories, files or subtests that need to be
        executed when a test case begins to run. The default pytest directory is
        ```pytest``. After the pytest run is finished, Twister will check if 
        the tst case passed or failed according to the pytest report.
        As an example, a list of valid pytest roots is presented below:
```